bpytop.py

```diff
@@ -3193,6 +3198,9 @@ def _get_nics(cls):
 			cls.nics.append(nic)
 		if not cls.nics: cls.nics = [""]
 		cls.nic = cls.nics[cls.nic_i]
+		if cls.net_iface: 
+                        cls.nic = cls.net_iface
```

Comment:
Should add `cls.nic_i = cls.nics.index(cls.nic)` here, for smooth switching behaviour.